packages/core-web/package.json

```diff
@@ -26,12 +26,14 @@
   "main": "./dist/index.js",
   "types": "./dist/index.d.ts",
   "dependencies": {
-    "@fedimint/fedimint-client-wasm-bundler": "workspace:*"
+    "@fedimint/fedimint-client-wasm-bundler": "workspace:*",
+    "secp256k1": "^5.0.0"
```

Comment:
it looks the crypto stuff is only being used in tests, so this dep should be moved to `devDependencies`